portfolio/src/main/webapp/index.html

```diff
@@ -129,18 +129,35 @@ <h5 class="card-title"> S.T.E.P. Internship Week 2 </h5>
                             <p class="card-text">This week I started with the individual projects set up for S.T.E.P Interns. I was also able to attend some official Google team meetings and some 1 on 1 meetings with my hosts. </p>
                         </div>
                         </br>
+                    </div>                   
+                </div>
+                </div>
+                </br>
+                
+            <div class="card">
+                <div class="card-header"> Comments </div>
+                <div class="card-body">
+                    <div class="card">
+                        <div class="card-body">
                         <form action="/comments" method="POST">
+                            <h6> Name </h6>
+                            <input type="text" name="name">
+                            </br>
                             <h6> Leave a comment letting us know what you think </h6>
                             <input type="text" name="comment">
                             </br>
-                            <input type="submit"  />
-                        </form> 
+                            <button onclick="getTime()"> Submit </button>
+                            <p id="test"> </p>
```

Comment:
is this <p> being used? if so, an id name other than test might help making its usage clearer.